utils/kata-manager.sh

```diff
@@ -10,6 +10,15 @@ set -o nounset
 set -o pipefail
 set -o errtrace
 
+#Added code :- simple checker, just sudo would require a 
+# passwword if sudo is not used before a restart
+if sudo modprobe vhost_vsock && sudo modprobe vhost_net; then
+  echo "VSOCK sockets and are supported"
+else
+  echo "Error: Host environment is too old to support VSOCK sockets"
```

Comment:
Could you send the message to standard error by using the `die()` function:

```suggestion
  die "Host environment is too old to support VSOCK sockets"
```